tests/CMakeLists.txt

```diff
@@ -1 +1,56 @@
+# Add an executable and its related test.
+#
+# The executable is always linked to 'kokkostools' and 'test_common'.
+#
+# The test environment always contains 'KOKKOS_TOOLS_LIBS' that is set as the path
+# to the target file of 'CONNECTOR_LIB'.
+#
+# Arguments:
+#   TARGET_NAME   : name of the test
+#   SOURCE_FILE   : source file, defaults to <TARGET_NAME>.cpp
+#   CONNECTOR_LIB : connector library target name under test
```

Comment:
Why not just call it `KOKKOS_TOOLS_LIBS` or `TOOLS_LIBS`?
It makes it self documented.
Also "connector" has been used to refer to some tools that inject user-defined annotations into vendor tools.  Clearly here that argument is intended for any tool.

Reply:
Indeed, you're right that "connector" is not a good choice.

So I've changed to `KOKKOS_TOOLS_LIBS` as you suggested, and it's now an optional argument :ballot_box_with_check: